articles/libraries/lock-android/v1/passwordless-magic-link.md

```diff
@@ -0,0 +1,168 @@
+---
+toc_title: Passwordless with Magic Link
+description: Passwordless with Magic Link with Lock Android
+---
+# Lock Android: Passwordless with Magic Link
```

Comment:
Same as the above comment. Passwordless is not deprecated but disabled. It might be enabled in the future (i don't know when). Warn them like here https://auth0.com/docs/libraries/lock-android/passwordless. BTW this other doc https://auth0.com/docs/libraries/lock-android/passwordless-magic-link also it's missing the warning.